docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,266 @@
+| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
+| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
+| Name               | Use letters, numbers, underscores, and hyphens. Do not use spaces.                                                                           |
+| Color Mode         | RGB                                                                                                                                               |
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the `manifest.json` file which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][22].
+
+Before you open a pull request, run the following command to catch any problems with your integration:
+```
+ddev validate all <INTEGRATION_NAME>
+```
+
+
+## Open a pull request
+
+Push up your feature branch and open a pull request that contains your integration tile's asset files (including images) in the [`marketplace`][18] or [`integrations-extras`][26] repository. After you've created your pull request, automatic checks will run to verify that your pull request is in good shape and contains all the required content to be updated.
+
+## Review process
+
+Once your pull request passes all checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace-review`, and `Datadog/documentation` teams provide suggestions and feedback on best practices.
+
+Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request. Contact the Marketplace team if you would like to preview the tile in your sandbox account. This allows you to validate and preview additional changes on the tile before your pull request is merged.
```

Comment:
```suggestion
Once you have addressed the feedback and re-requested reviews, these reviewers will approve your pull request. Contact the Marketplace team if you would like to preview your offering in your sandbox account. This allows you to validate and preview your offering before your pull request is merged.
```
Just a suggestion

Reply:
My concern is since this is focused on tile that it will confused partners by changing it to offering. @anawishnoff Thoughts on this? I changed the rest of the wording to match

Reply:
Keeping tile instead of offering